examples/app/sliders_applet/sliders_simple.py

```diff
@@ -0,0 +1,115 @@
+"""
+This file demonstrates a bokeh applet, which can be viewed directly
+on a bokeh-server. See the README.md file in this directory for
+instructions on running.
+"""
+
+import logging
+
+logging.basicConfig(level=logging.DEBUG)
```

Comment:
Mmmm.. it's there because of cut and paste fro the "main" example. That said, it seems there mostly for debugging reasons so I'm +1 on removing it.